collectfast/storage_extensions/__init__.py

```diff
@@ -0,0 +1,22 @@
+from django.utils.module_loading import import_string
+
+STORAGE_EXTENSIONS_MAP = {
+    # Maps the relevant `Storage` class to it corresponding `StorageExtensions` class
+    'django.core.files.storage.FileSystemStorage': 'collectfast.storage_extensions.file_system.FileSystemStorageExtensions',
```

Comment:
To implement #139, you would simply need to override this map value and the corresponding class with the desired `get_etag()` functionality